### ISSUE TITLE
Make the "x" in search bars an accessible button

### ISSUE THREAD

**Issue by witha-c** [cisagov/get.gov#301]
### Issue description

Some browsers add an "x", right justified, within the search bar. Users can click it to clear text, but it doesn't prompt other actions. Users can't tab/use keyboard nav to reach the x. 

Make the x a button within search bars that can be tabbed to that clears the search bar text. 

### Acceptance criteria

- [ ] an x appears in search bars, is right justified, and allows users to clear search field text
- [ ] in search views where users are searching in a table, it also resets the view
- [ ] the x behaves like a button and can be navigated to using a keyboard after the search text box and before the search button .

**Comment by AnnaGingle** [comment]
I don't think this is an accessibility issue because there is an alternate method to delete the form field content using the keyboard. It would be disorienting for a search to tab to the X button when they will be expecting a search button. When I first joined, I almost made a ticket for this and decided against it because it doesn't violate any WCAG criteria or follow the common search bar patterns.

**Comment by witha-c** [comment]
> @witha-c I don't think this is an accessibility issue because there is an alternate method to delete the form field content using the keyboard. It would be disorienting for a search to tab to the X button when they will be expecting a search button. When I first joined, I almost made a ticket for this and decided against it because it doesn't violate any WCAG criteria or follow the common search bar patterns. What do you think?

@AnnaGingle Good points - let's not do this!
